azure-pipelines/start-emulator.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Install AVD files
+# TODO: Do not hardcode Android SDK versions
+echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-28;google_apis;x86'
```

Comment:
Please move the emulator description into a constant to avoid duplication. Also, I would expect to retrieve the actual platform version from command line args/global env variables rather than have them hardcoded

Reply:
This was a cut-and-paste from the Microsoft website. I'm concerned about making edits to this file without breaking it.

Reply:
And also, the Mac OS agent has SDK versions 19 ~> 28 installed (see https://dev.azure.com/AppiumCI/Appium%20CI/_settings/agentqueues?queueId=17&_a=details) so you have to choose which SDK to run.

Reply:
I'll try though

Reply:
Aha, and they copy and pasted it from Travis and Travis copy and pasted it from who knows where ;)